x-pack/qa/reindex-tests-with-security/build.gradle

```diff
@@ -22,10 +21,11 @@ forbiddenPatterns {
 
 File caFile = project.file('src/test/resources/ssl/ca.p12')
 
-integTestCluster {
+testClusters.integTest {
+  distribution = "DEFAULT"
   // Whitelist reindexing from the local node so we can test it.
-  extraConfigFile 'http.key', project.projectDir.toPath().resolve('src/test/resources/ssl/http.key')
```

Comment:
👍  Thank you for this! Not sure why the usages of `Path` are littered in our build scripts but simply using `Project.file()` is so much more clear and concise.